packages/react-dom/src/__tests__/ReactTestUtilsActUnmockedScheduler-test.js

```diff
@@ -23,48 +24,43 @@ function clearLog() {
   }
 }
 
-function render(el, dom) {
-  ReactDOM.render(el, dom);
-}
-
-function unmount(dom) {
-  ReactDOM.unmountComponentAtNode(dom);
-}
-
 beforeEach(() => {
+  prevActGlobal = global.IS_REACT_ACT_ENVIRONMENT;
+  global.IS_REACT_ACT_ENVIRONMENT = true;
```

Comment:
This test is testing the unmocked scheduler (from the file name) so you can't use the internal act here.

Reply:
You should be able to use the public act here though.